### DISCUSSION TITLE
Glob Patterns fo Tags and the Filter Cheat Sheet

### DISCUSSION THREAD

**Discussion by mostlyfabulous** [community/community#26714] [Actions]
<p>I’ve been struggling to target tags that only bump the patch version for my workflow due to apparent limitations with the filters for it. I wanted to confirm, does filtering support full glob expressions and also, am I correctly understanding the negation character <code>!</code> only works at the beginning of the glob expression and not anywhere else in the pattern?</p>

**Comment by airtower-luna** [comment, marked as answer]
<p>There’s a description of the filter patterns in the docs, with a description of what exactly is supported: <a href="https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet" class="inline-onebox">Workflow syntax for GitHub Actions - GitHub Docs</a></p>
<p>And it confirms that the negating <code>!</code> must be at the start of the pattern.</p>

**Comment by mostlyfabulous** [comment]
<p>Thank you for confirming that, I think the main issue I had is that earlier on in the docs, it suggests that glob patterns in general should work:</p>
<blockquote>
<p>The <code>branches</code> , <code>branches-ignore</code> , <code>tags</code> , and <code>tags-ignore</code> keywords accept glob patterns that use characters like <code>*</code> , <code>**</code> , <code>+</code> , <code>?</code> , <code>!</code> and others to…</p>
</blockquote>
<p>From: <a href="https://on.push.%3Cbranches%7Ctags%7Cbranches-ignore%7Ctags-ignore%3E" rel="noopener nofollow ugc"><code>on.push.&lt;branches|tags|branches-ignore|tags-ignore&gt;</code></a></p>
<p>It would be helpful if they updated the docs to say that it accepts a GitHub Actions specific filter pattern instead of a glob pattern.</p>
